ruby-monday-blog/app/views/posts/_preview.html.erb

```diff
@@ -0,0 +1,22 @@
+<% hide_tags ||= false %>
```

Comment:
I put the hide_tags in there so that if you went to the 'Angular' tag-post index, it wouldn't show Angular in each post preview.  I'll have to refactor this.  Currently, it's hiding ALL tags in the tag index, which i don't think is intended